File: pulsar-skeleton/src/main/java/org/warps/pulsar/crawl/index/IndexingException.java
package org.warps.pulsar.crawl.index;

/**
 * Created by vincent on 16-8-1.
 */
public class IndexingException extends Exception {

    public IndexingException() {
        super();
    }

    public IndexingException(String message) {
        super(message);
    }

    public IndexingException(String message, Throwable cause) {
        super(message, cause);
    }

    public IndexingException(Throwable cause) {
        super(cause);
    }
}

File: pulsar-skeleton/src/main/kotlin/ai/platon/pulsar/skeleton/crawl/index/IndexingException.kt
package ai.platon.pulsar.skeleton.crawl.index

import java.lang.Exception

/**
 * Created by vincent on 16-8-1.
 */
class IndexingException : Exception {
    constructor() : super()
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
    constructor(cause: Throwable) : super(cause)
}
